### ISSUE TITLE
Demo - no controller found 

### ISSUE THREAD

**Issue by headwinds** [MikeFielden/The-MEAN-Stack#5]
I was able to follow your demo instructions and have it connect to mongo but I'm stuck...

Express server listening on port 1337 <-- nice port choice btw

When I go to: localhost:1337 

In the browser, I see:

Not Found 

In the console, I get an error message:

Controller not found 

I've looked over AppController.js and found that error message - I imagine its trying to do something with the only other controller: CatController 

Could you shed some light on what could be happening here?

**Comment by MikeFielden** [comment]
Did you change anything after you cloned it? 
The idea was to have you clone it then (assuming you have all the pre-req installed) just run 'mongod' and 'node server' and you should be gtg. 

Lemme know I'd love to help. :)

**Comment by headwinds** [comment]
nope. I'm pretty sure I followed the readme as described but I'll create a new directory and try again.

**Comment by MikeFielden** [comment]
Believe me its far from perfect just a jumping off point. 

I just cloned it and ran it with no problems.

**Comment by headwinds** [comment]
no worries it's cool - I tried again -- all tests pass but this time I'm getting a module loading error right away:

http://errors.angularjs.org/1.2.1/$injector/modulerr?p0=ngMeanStack&p1=Erro…alhost%3A1337%2Fmin%2FThe-MEAN-Stack-Boilerplate.min.v0.0.2.js%3A7%3A11551) The-MEAN-Stack-Boilerplate.min.v0.0.2.js:7

I'm very new to angular and I've basically been kicking the tires on few different mean stack github projects today - I liked your version because it included SCSS support :-D 

I also tried this repo today: https://github.com/linnovate/mean and got it up and running without problems. But that doesn't really help matter here just shows I can at least get something to work. hmmm....

**Comment by MikeFielden** [comment]
I havent tested this since Angular 1.2 came out... If it were anything Id bet its that. I can reproduce your error :/ 
Fudge.

**Comment by headwinds** [comment]
might be a good idea to use the exact version in the package.json or not... I see the benefits both ways... by using "latest", we get to squash new bugs as they pop up ;-D
